Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/TimeView/TimeViewPlugin.cpp

```diff
@@ -207,27 +227,50 @@ namespace EMStudio
         bodyLayout->setSpacing(0);
         innerWidget->setLayout(bodyLayout);
         bodyWidget->setWidget(innerWidget);
-        mainLayout->addWidget(bodyWidget, 2, 0, 1, 2);
 
         // Bottom-left
         m_trackHeaderWidget = new TrackHeaderWidget(this, m_dock);
         m_trackHeaderWidget->setFixedWidth(175);
         bodyLayout->addWidget(m_trackHeaderWidget);
 
         // Left
+        auto trackAndTrackDataWidget = new QWidget;
```

Comment:
Nit: `auto trackAndTrackDataWidget = new QWidget;` -> `auto* trackAndTrackDataWidget = new QWidget();`

Same for the other places below.